cpp/client/ct.cc

```diff
@@ -907,7 +909,7 @@ int GetSTH() {
   ct::SignedTreeHead sth;
   CHECK_EQ(AsyncLogClient::OK, client.GetSTH(&sth));
 
-  LogVerifier* verifier = GetLogVerifierFromFlags();
+  unique_ptr<LogVerifier> verifier(GetLogVerifierFromFlags());
```

Comment:
That one can't be const alas. I have to call reset() as not all paths use the verifier.